src/components/data_search/DatasetSearchTable.jsx

```diff
@@ -104,60 +104,48 @@ export const DatasetSearchTable = (props) => {
       queryChunks.push(...searchModifier);
     }
 
-    var filterQuery = {};
+    let filterQuery = {};
```

Comment:
I think `var` is not recommended now that we have `let` and `const`. The difference is that `var` has function scope, whereas `let` and `const` have block scope, which is usually what you want.